src/jvmMain/kotlin/io/beatmaps/api/review.kt

```diff
@@ -379,6 +379,17 @@ fun Route.reviewRoute() {
                             )
                             updateAlertCount(map.uploaderId.value)
                         }
+
+                        map.collaborators.values.filter { it.reviewAlerts }.forEach { singleCollaborator ->
+                            Alert.insert(
+                                "New review on a map you collaborated on",
+                                "@${sess.uniqueName} just reviewed a map you collaborated on #${toHexString(updateMapId)}: **${map.name}**.\n" +
+                                    "*\"${newText.replace(Regex("\n+"), " ").take(100)}...\"*",
+                                EAlertType.Review,
+                                singleCollaborator.id.value
```

Comment:
I'm now wondering why this is a loop at all when you can pass a list here

Reply:
For extra context.

If you call `Alert.insert(` 5 times it will create 5 alerts and join that to 5 recipients
If you call `Alert.insert(` once with a list of 5 users it will only create the alert once and join it to all 5

Reply:
(updateAlertCount also takes a list)

Reply:
True. I was under the impression I need it for updateAlertCount. But I see it has a list based API as well!